src/platform/sdl/sdl-log.c

```diff
@@ -0,0 +1,87 @@
+/* Copyright (c) 2013-2015 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "sdl-log.h"
+#include <mgba/core/core.h>
+#include <mgba/core/log.h>
+#include <mgba/core/thread.h>
+#include <mgba/core/config.h>
+
+static bool _logToStdout;
+static FILE* _logFile;
+
+struct mLogger _getLogger(bool logToStdout, bool logToFile, const char* logFile, int filterLevels);
+static void _mCoreLog(struct mLogger* logger, int category, enum mLogLevel level, const char* format, va_list args);
+
+bool getBoolConfig(struct mCore* core, const char* key, bool defaultValue)
+{
+	int value = 0;
+	bool searched = mCoreConfigGetValue(&core->config, key);
+	if(searched)
+	{
+		mCoreConfigGetIntValue(&core->config, key, &value);
+		return value != 0 ? true: false;
+	}
+	return defaultValue;
+}
```

Comment:
The existing convention is to use an int value called `fakeBool` and coerce to bool as needed. I may add an `mCoreConfigGetBoolValue` helper at some point, but not in this PR, and not a private one.

Reply:
Should be fixed in: https://github.com/mgba-emu/mgba/pull/2345/commits/02b165005bcfac32caf4ed53419a46ade0a6086f